src/ppx/application.cpp

```diff
@@ -998,6 +998,21 @@ void Application::DispatchConfig()
         mSettings.window.title = mSettings.appName;
     }
 
+    // ImGui doesn't support prebuilt command buffer.
+    if (mSettings.grfx.enablePreRecordCmd) {
+        PPX_LOG_WARN("Disable ImGui when enable prebuilt command buffer.");
+        mSettings.enableImGui = false;
+    }
+
+    if (mSettings.grfx.numFramesInFlight > mSettings.grfx.swapchain.imageCount) {
+        PPX_LOG_WARN("numFramesInFlight should not exceed image count in swapchain.");
```

Comment:
Also here, please say what action you're taking, i.e. "numFramesInFlight will be set to the swapchain image count"